src/layer/const.ts

```diff
@@ -99,3 +100,54 @@ export const SH_SERVICE_HOSTNAMES_V3: string[] = [
   'https://services-uswest2.sentinel-hub.com/',
   'https://creodias.sentinel-hub.com/',
 ];
+
+export type GetStatsAndHistogramParams = {
+  fromTime: Date;
+  toTime: Date;
+  crs: CRS;
+  resolution: number;
+  geometry: Polygon;
+  binAmount?: number;
+  evalscript?: string;
+};
+
+export type FisPayload = {
+  layer: string;
+  crs: CRS_IDS;
+  time: string;
+  resolution: number | string;
+  geometry?: string;
+  type?: HistogramType;
+  bins?: number;
+  evalscript?: string;
+  evalsource?: string;
+};
+
+export enum HistogramType {
+  EQUALFREQUENCY = 'EQUALFREQUENCY',
+  EQUIDISTANT = 'equidistant',
+  STREAMING = 'streaming',
```

Comment:
I would suggest removing these two options, since they are not supported (alternatively, we can also just comment them out - but maybe then they should be uppercased for consistency?), to avoid confusion.

Reply:
I made it uppercase, and commented it out